### PR TITLE
Update "Markdown Tutorial" deadlink (#Issue1457)

### PR DESCRIPTION
**Pull request by Jackbui96**
Gitraw Preview: https://rawgit.com/Jackbui96/Jackbui96.github.io/issue1457/#!pages/vi/vi-github-and-markdown.md

### CODE REVIEW

File: pages/vi/vi-github-and-markdown.md [hunk 1]
```diff
@@ -116,12 +116,12 @@ Make sure to include a rawgit link to your page in your pull request.
...
 ## Useful Links
 
-[GitHub: Writing and Formatting](https://help.github.com/categories/writing-on-github/) - A Github help page on how to format and write, along with working with saved replies.
-[Markdown](http://dynalon.github.io/mdwiki/#!quickstart.md) - A more descriptive Github guide on more markdown syntax.
-[Markdown Interactive Tutorial](http://www.markdowntutorial.com/lesson/1/) - An awesome interactive tutorial to experiment and learn more Markdown syntax with.
-[Mastering Markdown](https://guides.github.com/features/mastering-markdown/) - The official Github Guide for Markdown syntax
-[How to fork a repo](https://help.github.com/articles/fork-a-repo/) - A more in depth explanation about how and why we fork repositories from Github.
-[Comparison of markdown and html](http://thebridge.jp/en/wp/wp-content/uploads/2013/05/markdown-vs-html.png) - The general comparison of markdown and html.
-[Other helpful links and videos](vi-faq.md#Helpful_Links)
+[GitHub: Writing and Formatting](https://help.github.com/categories/writing-on-github/) - A Github help page on how to format and write, along with working with saved replies.
```
Comment (xyb994) [inline pages/vi/vi-github-and-markdown.md]: Please watch out for trailing whitespace. These make it looks like you made more change than you need to.

Comment (Jackbui96) [reply pages/vi/vi-github-and-markdown.md]: @xyb994 I was informed that by using the 2 white spaces at the end of the sentence I can do the soft break, based on markdowntutorial.com. That is the reason why I am adding the white spaces at the end of the sentence, as I am wondering if the method works or not.

Source: https://www.markdowntutorial.com/lesson/7/

If you think that the soft break is unnecessary, then I will stop adding the white spaces and will re-edit my PR.

Comment (xyb994) [reply pages/vi/vi-github-and-markdown.md]: @Jackbui96 My bad, I didn't know about that feature and didn't realize what you were trying to achieve. Thanks for explaining your intention for that. Looks like it does fix the problem in GitHub Flavor's Markdown ([master](https://github.com/open-learning-exchange/open-learning-exchange.github.io/blob/master/pages/vi/vi-github-and-markdown.md), [Jackbui96:issue1457](https://github.com/Jackbui96/Jackbui96.github.io/blob/f3458b14a1805d7829015d354f411bfa0aa1ae10/pages/vi/vi-github-and-markdown.md)). But it looks like MDwiki already have this soft break even without two white spaces ([master](https://open-learning-exchange.github.io/#!./pages/vi/vi-github-and-markdown.md#Useful_Links), [Jackbui96:issue1457](https://rawgit.com/Jackbui96/Jackbui96.github.io/issue1457/#!./pages/vi/vi-github-and-markdown.md#Useful_Links)).

This comes back to [a note we just added](https://github.com/open-learning-exchange/open-learning-exchange.github.io/commit/7f71e0a82af7c9ffcf34802c6bd3f62635b229ae#diff-5b72c36d6c2ea1941fbdaa2e0edc04a7R102), there are no standard Marldown syntax so this is a little hard to deal with...

Comment (Jackbui96) [reply pages/vi/vi-github-and-markdown.md]: @xyb994 Can I just add a comment explaining why there are white spaces at the end? If someone was to edit the file in the future, they will know why there are white spaces at the end of the sentences. I am thinking that by adding comment, it is also consider making the code looks "cleaner". What do you think about this approach?

**Comment by xyb994** [comment]
Closing this, see https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues/1532#issuecomment-359199586

**Pull request closed without merging**